spec/models/order_spec.rb

```diff
@@ -11,4 +11,30 @@
     it { should have_many :order_items }
     it { should have_many(:items).through(:order_items) }
   end
+
+  describe 'instance methods' do
+    it '.create_order_items' do
+      distributor = Distributor.create!(name: 'RNDC',
+                                        address: '3319 Arapahoe st, Denver, CO',
+                                        code: 'CODE1234',
+                                        api_key: 'jgn983hy48thw9begh98h4539h4',
+                                        password: 'password'
+                                        )
+# THIS TEST WILL FAIL UPON MERGING TO MASTER DUE TO Business needing a representative.
+      business = create(:business, api_key: '444444444', distributor: distributor)
```

Comment:
Factory bot will set up an api_key for any business automatically, we don't need to do this